src/locales/index.ts

```diff
@@ -1,7 +1,7 @@
 import { en } from './en'
 
 // If you wish to define additional locales, add them here separated by `|`
-export type DefinedLocales = 'en'
+export type DefinedLocales = 'en|de'
 
 export const locales: Record<DefinedLocales, typeof en> = {
   en
```

Comment:
`de` (from the suggested import above) should be included in the locales record
```suggestion
  en,
  de,
```